theano/gpuarray/dnn.py

```diff
@@ -361,7 +361,7 @@ def do_constant_folding(self, node):
         return False
 
     def __init__(self, border_mode, subsample=(1, 1), conv_mode='conv',
-                 precision="float32"):
+                 precision="float32", precision="float64"):
```

Comment:
Also, I don't think this is the right place to fix it. `float32` is a reasonable default.
What we should do is check where `gpu_dnn_conv_desc` is called, and explicitly call it with the dtype corresponding to the precision of that particular convolution.